templates/drupal-fix-permissions.sh.erb

```diff
@@ -72,7 +74,30 @@ if [ -z "${drupal_user}" ] || [[ $(id -un "${drupal_user}" 2> /dev/null) != "${d
   exit 1
 fi
 
-cd $drupal_path
+printf "Checking Drupal version at "${drupal_path}"... ";
+drupal_version=$(drush status --root=${drupal_path} | grep 'Drupal version' | awk -F: '{print $2}' | awk -F. '{print $1}' | tr -d [:space:])  
```

Comment:
I'd just use ``drush status --field=drupal-version``

Give it a test in drush 8 too

Reply:
Doesn't work in drush 8...  :(

Reply:
See if there's something equivalent

Reply:
Yes, I am looking for something like that.

Reply:
Closest I can get is: `drush core-status version` which outputs:

```
 Drupal version : 8.6.14 
 Drush version  : 9.5.2
```